src/graph/table.rs

```diff
@@ -32,15 +33,26 @@ pub(crate) struct TableArgs {
     #[arg()]
     records: PathBuf,
 
-    /// Path to breakend records in BCF format
+    /// Path to reference
```

Comment:
We could even make this more descriptive:
```suggestion
    /// Path to reference file in fasta format
```
And does `bio::io::fasta::Reader()` handle gzipped files? From a quick look, I don't see any functionality for this in the code, but maybe this is some levels further down (e.g. in `std::fs`)?